src/redux/books/books.js

```diff
@@ -31,3 +31,32 @@ export const fetchBooksSuccess = (payload) => ({
   type: FETCH_SUCCESS,
   payload,
 });
+export const fetchBooks = () => async (dispatch) => {
+  const response = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/WsQJJJQ05Rg6MbRzcWLl/books');
+  const bookData = await response.json();
+  const books = Object.entries(bookData).map(([key, value]) => ({
+    item_id: key,
+    title: {
+      title: value[0].title.title,
+      author: value[0].title.author,
+    },
+    category: value[0].category,
+  }));
+  dispatch(fetchBooksSuccess(books));
+};
```

Comment:
- [ ] You are required to use `createAsyncThunk` for your API calls. Kindly refactor your fetch method to use that. You may want to look at the link provided in the guidelines for how to use it: [https://redux-toolkit.js.org/api/createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk)